icssc/AntAlmanac
# Show all past enrollment graphs

## Summary
- The user can navigate back and forth between all the past enrollment graphs for a specific course

![image](https://github.com/icssc/AntAlmanac/assets/78244965/1ffa94d4-0181-4f21-b221-1135be441bfd)


## TODO
- [x] Show all past enrollment graphs
- [x] Improve the design for mobile

## Test Plan
- Verify this works for classes in the most recent quarter (e.g. CS 143A).
- Verify this works for classes that haven't been offered very recently (e.g. CS 122D from Spring 2023).
- Make sure the modal looks good on mobile and light/dark mode.

## Issues

Closes #830 

## Future Follow-up
- I tried to add an instructor filter, but Material UI has problems when we add a select menu to a popper. We can tackle this later. Or, we can just not implement it because it would clutter the popper, and the user can go to ZotTracker for more advanced searching.
